src/curated-corpus/components/ApprovedItemForm/ApprovedItemForm.tsx

```diff
@@ -106,6 +117,34 @@ export const ApprovedItemForm: React.FC<
     formik.setFieldValue('imageUrl', url);
   };
 
+  // state variable to store and set Open Graph excerpt
+  const [ogExcerpt, setOgExcerpt] = useState<string>('');
+
+  /**
+   * This calls the query on component mount. This isn't ideal because the editors might not even want the OG excerpt so we
```

Comment:
If the web page is end-user facing 💯 . In this case -- where the usage of this page is low and only for curators -- it doesn't read to me as a major problem. (If anything it means that the editors can toggle and choose and that'll be performant, right?)